vnext/Microsoft.ReactNative/Modules/DeviceInfoModule.cpp

```diff
@@ -45,15 +49,40 @@ void DeviceInfoHolder::InitDeviceInfoHolder(
           });
     } else {
       assert(react::uwp::IsXamlIsland());
-      // TODO: WinUI 3 Islands - set up a listener for window size changed
+      // This is either a WinUI 3 island or a system XAML island
```

Comment:
>// This is either a WinUI 3 island or a system XAML island [](start = 6, length = 58)

I have a fix for the background activation that changes the same code. In the case of the background activation, the current window is also null and I had to work around it. My fix works only for the system XAML. I think I must submit my change on top of yours to make sure that it works with WinUI 3.

Reply:
that would be great : )